health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,325 @@
+const MapViewPopupWrapper = () => {
+  const [showPopup, setShowPopup] = useState(false);
+
+  return (
+    <>
+      <Button
+        label={"click"}
+        onClick={() => {
+          setShowPopup(true);
+        }}
+        type="button"
+      ></Button>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)